### PR TITLE
Add variant G070RB

### PR DESCRIPTION
**Pull request by MrZloHex**
## Pull Request template

**Summary**

Adding new variant for Generic `G070RBT`

This PR fixes/implements the following **bugs/features**

* [ ] New variant for NUCLEO-G070RB
* [X] Add variant for bare STM32G070RB

New variant will be supported by stm32duino

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,6 +301,7 @@ User can add a STM32 based board following this [wiki](https://github.com/stm32d
 | :green_heart: | STM32G031K4<br>STM32G031K6<br>STM32G031K8 | Generic Board | *2.0.0* |  |
 | :green_heart: | STM32G041J6 | Generic Board | *2.0.0* |  |
 | :green_heart: | STM32G041K6<br>STM32G041K8 | Generic Board | *2.0.0* |  |
+| :green_heart: | STM32G070RB | Generic Board | *0.1.0* |  |
```
Comment (ABOSTM) [inline README.md:304]:
```suggestion
| ::yellow_heart:: | STM32G070RB | Generic Board | *2.3.0* |  |
```

File: variants/STM32G0xx/G070RBT/generic_clock.c [hunk 2]
```diff
@@ -20,8 +20,41 @@
   */
 WEAK void SystemClock_Config(void)
 {
-  /* SystemClock_Config can be generated by STM32CubeMX */
-#warning "SystemClock_Config() is empty. Default clock at reset is used."
+  RCC_OscInitTypeDef RCC_OscInitStruct = {0};
+  RCC_ClkInitTypeDef RCC_ClkInitStruct = {0};
```
Comment (ABOSTM) [inline variants/STM32G0xx/G070RBT/generic_clock.c:24]: With compiler we used, assignation to 0 on such heterogenous structure causes compiler warning,
(and are not required)
```suggestion
  RCC_OscInitTypeDef RCC_OscInitStruct = {};
  RCC_ClkInitTypeDef RCC_ClkInitStruct = {};
```

**Review by ABOSTM — CHANGES_REQUESTED** [review]
Hi @MrZloHex,
Thanks for this PR.
Few inline comment.
Also could you run Astyle on generic_clock.c:
https://github.com/stm32duino/wiki/wiki/Astyle

**Comment by ABOSTM** [comment]
By the way, you can squash the whole in a single commit.

**Comment by ABOSTM** [comment]
Thanks for the update, could you run Astyle on generic_clock.c:
https://github.com/stm32duino/wiki/wiki/Astyle
This will fix "STM32 Core Continuous Integration" check,
and by the way squash the whole in a single commit.

**Comment by MrZloHex** [comment]
Ok

**Comment by fpistm** [comment]
Any update on this PR ?

**Comment by ABOSTM** [comment]
This PR is superseded by #1638

**Pull request closed without merging**